package.json

```diff
@@ -9,11 +9,19 @@
   },
   "scripts": {
     "start": "npm run dev",
+    "start:ci:english": "cross-env ELEVENTY_ENV=ci LOCALE_FOR_UI=english npm run dev",
+    "start:ci:chinese": "cross-env ELEVENTY_ENV=ci LOCALE_FOR_UI=chinese npm run dev",
+    "start:ci:espanol": "cross-env ELEVENTY_ENV=ci LOCALE_FOR_UI=espanol npm run dev",
```

Comment:
```suggestion
    "start:ci:english": "cross-env ELEVENTY_ENV=ci LOCALE_FOR_UI=english LOCALE_FOR_GHOST=english npm run dev",
    "start:ci:chinese": "cross-env ELEVENTY_ENV=ci LOCALE_FOR_UI=chinese LOCALE_FOR_GHOST=chinese npm run dev",
    "start:ci:espanol": "cross-env ELEVENTY_ENV=ci LOCALE_FOR_UI=espanol LOCALE_FOR_GHOST=espanol npm run dev",
```

This might fix it.

Reply:
> It does look like LOCALE_FOR_GHOST and LOCALE_FOR_UI are mostly the same. Are they sometimes different? If not, we can get rid of one.

They are always going to be the same in production like deployments. However we do need them to different sometimes during local builds because we have seeded articles in say English and would want to test i18n in a different language

Reply:
Thanks @ojeytonwilliams. I'll give it a shot.